### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by RipDip**
Add IntelliJ IDEA integration and Mac OS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,15 @@ TODO
   - Click `Finish`
 
 ### IntelliJ IDEA integration
-- TODO
+- Project integration:
+	- If no project is opened in IntelliJ IDEA, then Left click on `Open`. Otherwise Left click on `File` choose `Open`
```
Comment (cagix) [inline README.md:65]: "Left click" => "left click"

File: README.md [hunk 2]
```diff
@@ -61,7 +61,15 @@ TODO
...
+	- Choose the project root directory
+	- Choose `Gradle project` or `Maven project`
+	- Choose `Trust Project`
+	- Choose `This Windows` or `new Windows`
+	
+### Mac OS
+- If you use Mac OS, you must add the VM-Argument `-XstartOnFirstThread`
```
Comment (cagix) [inline README.md:72]: is this specific to intellij? or is this generally required when using our dungeon on macos?

**Review by cagix — CHANGES_REQUESTED** [review]
👍some minor suggestions

File: README.md [hunk 3]
```diff
@@ -61,7 +61,15 @@ TODO
   - Click `Finish`
 
 ### IntelliJ IDEA integration
-- TODO
+- Project integration:
+	- If no project is opened in IntelliJ IDEA, then Left click on `Open`. Otherwise Left click on `File` choose `Open`
+	- Choose the project root directory
+	- Choose `Gradle project` or `Maven project`
+	- Choose `Trust Project`
+	- Choose `This Windows` or `new Windows`
```
Comment (AMatutat) [inline README.md:69]: realy WindowS?

File: README.md (hunk 2 above)
Comment (AMatutat) [reply README.md:72]: where to add this?

**Review by AMatutat — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (cagix) [reply README.md:72]: not sure, maybe we introduce a "odds&ends" sections?

File: README.md [hunk 4]
```diff
@@ -61,7 +61,32 @@ TODO
...
-- TODO
+- Project integration:
+	- If no project is opened in IntelliJ IDEA, then left click on `Open`. Otherwise left click on `File` choose `Open`
+	- Choose the project root directory
+	- Choose `Gradle project` or `Maven project`
+	- Choose `Trust Project`
+	- Choose `This Window` or `new Window`
+	
+### Mac OS
+- If you use Mac OS, you must add the VM-Argument `-XstartOnFirstThread` in your IDE.
```
Comment (cagix) [inline README.md:72]: isn't the correct term "macOS"?

File: README.md [hunk 5]
```diff
@@ -61,7 +61,32 @@ TODO
...
+		- Left click on `Arguments`
+		- In the `VM arguments:` add `-XstartOnFirstThread`
+		- Click `Run`
+	- In IntelliJ IDEA
+		- Left click on `Edit Configurations...`
+		- Left click on `+`
+		- Choose `Application`
+		- Set a name
+		- Left click on `Modify options` and choose `Add VM options`
+		- By `VM options` you add `-XstartOnFirstThread`
```
Comment (cagix) [inline README.md:85]: "By" sounds kind of strange. Maybe "For" or "At"?

File: README.md [hunk 6]
```diff
@@ -61,7 +61,32 @@ TODO
...
+		- By `Main Class` you add the main Class from the project root directory
```
Comment (cagix) [inline README.md:86]: dito

File: README.md [hunk 7]
```diff
@@ -61,7 +61,32 @@ TODO
...
+- Project integration:
+	- If no project is opened in IntelliJ IDEA, then left click on `Open`. Otherwise left click on `File` choose `Open`
+	- Choose the project root directory
+	- Choose `Gradle project` or `Maven project`
+	- Choose `Trust Project`
+	- Choose `This Window` or `new Window`
+	
+### Mac OS
+- If you use Mac OS, you must add the VM-Argument `-XstartOnFirstThread` in your IDE.
+	- In Eclipse:
```
Comment (cagix) [inline README.md:73]: "For" Eclipse?

File: README.md [hunk 8]
```diff
@@ -61,7 +61,32 @@ TODO
...
+		- Right click in project explorer and choose `Run As`
+		- Choose `Run Configurations...`
+		- Left click on `Arguments`
+		- In the `VM arguments:` add `-XstartOnFirstThread`
+		- Click `Run`
+	- In IntelliJ IDEA
```
Comment (cagix) [inline README.md:79]: dito

Comment (cagix) [reply README.md:79]: also: end line with ":"?

**Review by cagix — CHANGES_REQUESTED** [review]
🦖

File: README.md [hunk 9]
```diff
@@ -61,7 +61,32 @@ TODO
...
 ### IntelliJ IDEA integration
-- TODO
+- Project integration:
+	- If no project is opened in IntelliJ IDEA, then left click on `Open`. Otherwise left click on `File` choose `Open`
+	- Choose the project root directory
+	- Choose `Gradle project` or `Maven project`
+	- Choose `Trust Project`
+	- Choose `This Window` or `new Window`
+	
+### Mac OS
```
Comment (cagix) [inline README.md:71]: still "Mac OS"? => "macOS" :)

**Review by cagix — CHANGES_REQUESTED** [review]
one last "Mac OS" ... except this it looks fine to me.

**Review by cagix — COMMENTED** [review]
lgtm 👍

**Review by AMatutat — APPROVED** [review]

**Pull request merged**